server/src/ScriptRunner/Scripts/index/1_insert_new_cities.sql

```diff
@@ -0,0 +1,18 @@
+insert into public.cities (id,display_name,county_id,desk_id,desk_mapping_technique) values('s1024','מעבר תרקומיא',66,801,1) ON CONFLICT DO NOTHING;
+insert into public.cities (id,display_name,county_id,desk_id,desk_mapping_technique) values('s1025','מעבר מיתר',66,801,1) ON CONFLICT DO NOTHING;
+insert into public.cities (id,display_name,county_id,desk_id,desk_mapping_technique) values('s1026','מעבר ל"ה',66,801,1) ON CONFLICT DO NOTHING;
+insert into public.cities (id,display_name,county_id,desk_id,desk_mapping_technique) values('s1027','מעבר אייל',66,801,1) ON CONFLICT DO NOTHING;
+insert into public.cities (id,display_name,county_id,desk_id,desk_mapping_technique) values('s1028','מעבר רחל',66,801,1) ON CONFLICT DO NOTHING;
+insert into public.cities (id,display_name,county_id,desk_id,desk_mapping_technique) values('s1029','מעבר קלנדיה',66,801,1) ON CONFLICT DO NOTHING;
+insert into public.cities (id,display_name,county_id,desk_id,desk_mapping_technique) values('s1030','מעבר שער-אפריים',66,801,1) ON CONFLICT DO NOTHING;
+insert into public.cities (id,display_name,county_id,desk_id,desk_mapping_technique) values('s1031','מעבר ריחן',66,801,1) ON CONFLICT DO NOTHING;
+insert into public.cities (id,display_name,county_id,desk_id,desk_mapping_technique) values('s1032','מעבר גלבוע',66,801,1) ON CONFLICT DO NOTHING;
+insert into public.cities (id,display_name,county_id,desk_id,desk_mapping_technique) values('s1033','מעבר נהר הירדן',66,802,1) ON CONFLICT DO NOTHING;
+insert into public.cities (id,display_name,county_id,desk_id,desk_mapping_technique) values('s1034','מעבר טאבה',66,802,1) ON CONFLICT DO NOTHING;
+insert into public.cities (id,display_name,county_id,desk_id,desk_mapping_technique) values('s1035','מעבר רבין',66,802,1) ON CONFLICT DO NOTHING;
+insert into public.cities (id,display_name,county_id,desk_id,desk_mapping_technique) values('s1036','מעבר ימי אשקלון',66,803,1) ON CONFLICT DO NOTHING;
+insert into public.cities (id,display_name,county_id,desk_id,desk_mapping_technique) values('s1037','מעבר ימי אשדוד',66,803,1) ON CONFLICT DO NOTHING;
+insert into public.cities (id,display_name,county_id,desk_id,desk_mapping_technique) values('s1038','מעבר ימי אילת',66,803,1) ON CONFLICT DO NOTHING;
+insert into public.cities (id,display_name,county_id,desk_id,desk_mapping_technique) values('s1039','מעבר אוירי רמון',66,804,1) ON CONFLICT DO NOTHING;
+insert into public.cities (id,display_name,county_id,desk_id,desk_mapping_technique) values('s1040','מעבר נתב"ג',66,804,1) ON CONFLICT DO NOTHING;
```

Comment:
BTW next time you can use VALUES with an array

Reply:
👴